include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -834,7 +834,7 @@ __parallel_scan_copy(sycl::queue& __q, _InRng&& __in_rng, _OutRng&& __out_rng, _
 
     _Assigner __assign_op;
     _ReduceOp __reduce_op;
-    _DataAcc __get_data_op;
+    _DataAcc __get_data_op{};
```

Comment:
Why is initialization needed for `__get_data_op` but not needed for three other variables around it?

Reply:
If it is a false positive by coverity, we can as well mark it such.
The question is, as usual - are we sure these types are always default-constructible?

Reply:
To add more details, the modified functors have no members, except for two having `std::plus` and `dpl::identity`. I cannot prove that `std::plus` is default constructible.

Reply:
If these classes do not depend on user-provided functions/types and we are sure these are default-constructible, then it's fine (though I wonder then if the variables are even needed).